batch/batch/driver/gce.py

```diff
@@ -184,4 +184,4 @@ async def delete_orphaned_disks(self):
                     log.exception(f'error while deleting orphaned disk {disk_name}')
 
     async def delete_orphaned_disks_loop(self):
-        await periodically_call(15, self.delete_orphaned_disks)
+        await periodically_call(60, self.delete_orphaned_disks)
```

Comment:
I made this 60. We're making too many API calls listing disks for each zone. I'm open to making it 15, but I don't think it matters whether it's 15 or 60 and is better to conserve api calls.